src/modules/imports/import.rs

```diff
@@ -69,12 +70,22 @@ impl Import {
     }
 
     fn resolve_import(&mut self, meta: &ParserMetadata) -> Result<String, Failure> {
-        match fs::read_to_string(self.path.value.clone()) {
-            Ok(content) => Ok(content),
-            Err(err) => error!(meta, self.token_path.clone() => {
-                message: format!("Could not read file '{}'", self.path.value),
-                comment: err.to_string()
-            })
+        if self.path.value.starts_with("std") {
+            match stdlib::resolve(&self.path.value.replace("std/", "")) {
+                Some(v) => Ok(v),
+                None => error!(meta, self.token_path.clone() => {
+                    message: format!("Std module does not exist: {}", self.path.value),
+                               comment: String::new()
+                })
```

Comment:
We can use the simpler version if no comments are provided

```suggestion
                None => error!(meta, self.token_path.clone(),
                    format!("Standard library module '{}' does not exist", self.path.value))
```